meditate/templates/home.html

```diff
@@ -429,26 +451,28 @@ <h2 class="wsite-content-title">Contact Jivan<br /></h2>
 
     };
 
-    (function(d, s, id){
+    (function (d, s, id) {
         var js, fjs = d.getElementsByTagName(s)[0];
-        if (d.getElementById(id)) {return;}
+        if (d.getElementById(id)) { return; }
         js = d.createElement(s); js.id = id;
-        js.src = "//connect.facebook.net/"+_W.facebookLocale+"/sdk.js";
+        js.src = "//connect.facebook.net/" + _W.facebookLocale + "/sdk.js";
         fjs.parentNode.insertBefore(js, fjs);
     }(document, 'script', 'facebook-jssdk'));
+
 </script>
 <script type='text/javascript'>
-    (function() {
+    (function () {
         function j(src, id) {
             var s = document.createElement('script');
             s.type = 'text/javascript';
             s.async = true;
             s.src = src;
-            if(id) {s.id=id};
+            if (id) { s.id = id };
             document.getElementsByTagName('head')[0].appendChild(s);
         }
-        j('//platform.twi'+'tter.com/widgets.js');
+        j('//platform.twi' + 'tter.com/widgets.js');
     })()
+
 </script>
 
-{% endblock %}
+{% endblock %}
```

Comment:
There's an awful lot of javascript in here.  Better to refactor it into a main.js.  Is all of this main-specific or does some of it belong to base?

Reply:
Yes, after comparing it from the live site, I found that some of them are common for all the pages. However, their functionalities are difficult to understand since they are all 3rd party prebuilt functions. Hence, I am giving them a common naming sequence which you will see in the next commit.